source/Ubuntu-B.ufo/lib.plist

```diff
@@ -1471,113 +1263,11 @@
 			<string>uni1FBC.alt</string>
 			<string>uni1FCC.alt</string>
 			<string>uni1FFC.alt</string>
-			<string>glyph1467</string>
-			<string>glyph1468</string>
-			<string>glyph1469</string>
-			<string>glyph1470</string>
-			<string>glyph1471</string>
-			<string>SF540000</string>
-			<string>SF530000</string>
-			<string>SF190000</string>
-			<string>SF360000</string>
-			<string>SF450000</string>
-			<string>SF280000</string>
-			<string>SF500000</string>
-			<string>SF470000</string>
-			<string>SF220000</string>
-			<string>SF510000</string>
-			<string>SF240000</string>
-			<string>SF440000</string>
-			<string>SF230000</string>
-			<string>SF420000</string>
-			<string>SF400000</string>
-			<string>SF260000</string>
-			<string>SF380000</string>
-			<string>SF410000</string>
-			<string>SF250000</string>
-			<string>SF390000</string>
-			<string>SF430000</string>
-			<string>SF200000</string>
-			<string>SF370000</string>
-			<string>SF460000</string>
-			<string>SF270000</string>
-			<string>SF490000</string>
-			<string>SF480000</string>
-			<string>SF210000</string>
-			<string>SF520000</string>
-			<string>block</string>
-			<string>ltshade</string>
-			<string>shade</string>
-			<string>dkshade</string>
```

Comment:
This is also removing `block`, `ltshade`, `shade`, `dkshade`.

Reply:
> This is also removing block, ltshade, shade, dkshade.

probably because these glyphs are only contained in the monospaced fonts, but not in the normal Bold and sibiling fonts.